tests/system_tests_grpc.py

```diff
@@ -0,0 +1,215 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import os, sys
+from time import sleep, ctime
+import system_test
+from system_test import TestCase, Qdrouterd, QdManager, Process, SkipIfNeeded, TIMEOUT
+from subprocess import PIPE
+try:
+    import grpc
+    import friendship_server as fs
+    from friendship_pb2_grpc import FriendshipStub
+    from friendship_pb2 import Person, FriendshipRequest, PersonEmail
+except ImportError:
+    pass
+
+
+def skip_test():
+    """
+    If grpc cannot be imported, test must be skipped
+    :return:
+    """
```

Comment:
Forgot to mention. Dispatch can be compiled without support for HTTP2. Tests have to be skipped in that case too. Probably easiest to do this in CMake, than here. Although the router should have some way of checking for features like this...

Reply:
Created https://github.com/apache/qpid-dispatch/pull/1006 to fix this.